README.md

```diff
@@ -114,7 +114,7 @@ ctest --test-dir build
 
 ## Licensing
 
-CXX-Qt is Copyright (C) 2022, Klarälvdalens Datakonsult AB, and is available under
+CXX-Qt is Copyright (C) 2023, Klarälvdalens Datakonsult AB, and is available under
```

Comment:
Usually we have the first year in the copyright date or you specify a range or list all the years. Having just the latest year is a bit odd.

Personally I'd like to receive legal advise if the copyright year is even required anymore and if git commit dates can be used instead.